docs/libcurl/opts/CURLOPT_HTTPPOST.3

```diff
@@ -32,6 +32,8 @@ CURLOPT_HTTPPOST \- multipart formpost content
 CURLcode curl_easy_setopt(CURL *handle, CURLOPT_HTTPPOST,
                           struct curl_httppost *formpost);
 .SH DESCRIPTION
+\fBThis function is deprecated.\fP Use \fICURLOPT_MIMEPOST(3)\fP instead.
```

Comment:
Isn't it an option rather than a function ?

Reply:
Agreed!

Reply:
> Isn't it an option rather than a function ?

Oops was a c&p, fixed now

~~~diff
-\fBThis function is deprecated.\fP Use \fICURLOPT_MIMEPOST(3)\fP instead.
+\fBThis option is deprecated.\fP Use \fICURLOPT_MIMEPOST(3)\fP instead.
~~~